macros/WebExtBrowserCompat.ejs

```diff
@@ -42,7 +42,7 @@ function tableFromArgument(path) {
   const manifestDataUrl = "https://raw.githubusercontent.com/mdn/browser-compat-data/master/webextensions/manifest-keys.json";
```

Comment:
Is renaming this (https://raw.githubusercontent.com/mdn/browser-compat-data/master/webextensions/manifest-keys.json) is on the eventual plan as well?